js/data/locale/tg/list.json

```diff
@@ -0,0 +1,16 @@
+{
+    "or": {
+        "medium": {
+            "2": "{0} ё {1}",
+            "end": "{0}, ё {1}"
```

Comment:
This needs to be hard coded in the genlist.js tool so that the next time we run it, we don't overwrite these.